README.rst

```diff
@@ -28,4 +28,23 @@ The plugin can be installed with given `requirements.sh <requirements.sh>`_ or t
 
            sudo apt install -y sense-hat
 
-Once that's done, it's probably a good idea to reboot your Pi to let the changes propagate.
+
+Also it is required to enable I2C on the RPi, please do the following steps:
+1) sudo raspi-config
+2) Choose Interfacing options
+3) Choose P5I2C option Enable/Disable automatic loading of I2C Kernel module
```

Comment:
We should add a space here. Or may be just a link to install_notes.txt to avoid duplicates.